app/Main.hs

```diff
@@ -461,7 +484,16 @@ verifyPackage db paths name = do
   echoT ("Verifying package " <> runPackageName name)
   dependencies <- map fst <$> getTransitiveDeps db [name]
   let srcGlobs = map (pathToTextUnsafe . (</> ("src" </> "**" </> "*.purs")) . dirFor) dependencies
-  procs "purs" ("compile" : srcGlobs) empty
+  purs <- pursCmd
+  run purs srcGlobs
+  where
+    run :: MonadIO io => Text -> [Text] -> io ()
+    run "purs.cmd" globs =
```

Comment:
I think this special case should be fused into the function which finds the executable.

i.e. one function which finds `purs` and runs it with some arguments.

Reply:
So, a `runPurs` command, looks up the executable and does the right thing with the globs to execute it?

Reply:
Yes, I think so.

Reply:
ok... Let me try it and if it's not what you had in mind, I'll be glad to change it.